Toolchain/BuildIt.sh

```diff
@@ -75,10 +75,16 @@ GCC_NAME="gcc-$GCC_VERSION"
 GCC_PKG="${GCC_NAME}.tar.gz"
 GCC_BASE_URL="http://ftp.gnu.org/gnu/gcc"
 
+SYSTEM_NAME="$(uname -s)"
```

Comment:
this is not used anywhere? Caching the result here is nice as it's not going to change, can you replace the below uname calls with this variable?

Reply:
Ah yes, I was planning to do that but then I forgot. Thanks, will do.

Reply:
There we go. Also, I've tested it again on Linux and macOS and it seems to work without problems.